app/crud/sync_db/tile_cache_assets.py

```diff
@@ -134,7 +134,7 @@ def get_attributes(dataset: str, version: str, asset_type: str) -> List[Dict[str
         #  needs changes to data-api to assure dynamic vector tile caches
         #  also have the implementation parameter in the creation options
         if tile_cache["dataset"] == dataset and tile_cache["version"] == version:
-            return tile_cache["fields"]
+            return tile_cache["metadata"]["fields"]
```

Comment:
Looking at the query, I think this should be `tile_cache["fields"]`

Reply:
Me too: https://github.com/wri/gfw-tile-cache/blob/9c0074ce7f1a7ab202e688ebaaeafcb5d9cce920/app/crud/sync_db/tile_cache_assets.py#L26
Are you sure about this change, @jterry64 ?

Reply:
Ah ok, so then I'd need to change `assets.fields as fields` to `asset.metadata->fields as fields` and revert the thing above? `fields` is definitely under `metadata` now

Reply:
That sounds correct to me!